src/main/java/seedu/address/commons/core/ProfilePicturesFolder.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.commons.core;
+
+/**
+ * Contains path to user's folder that contains profile pictures of all clients.
+ */
+public class ProfilePicturesFolder {
+
+    private static String profilePicsFolderPath = "C:/Users/acer/Desktop/SE/profilepic/";
```

Comment:
This default file path will cause a problem because it does not exist for the rest of us. Please move the profile picture into  `../resources/images` folder and change the file path accordingly

Reply:
For the picture above:  Please move the profile picture into  `../resources/images` folder and change the file path in the code accordingly

Reply:
Noted